agenta-backend/agenta_backend/routers/app_router.py

```diff
@@ -27,6 +25,9 @@
 )
 from agenta_backend.models import converters
 
+if os.environ["FEATURE_FLAG"] not in ["ee"]:
```

Comment:
I don't think this is the right solution. Instead I would refactor this line ([229
](https://github.com/Agenta-AI/agenta/blob/b544089328132ed6fac67b5ffea638a2c312af2e/agenta-backend/agenta_backend/routers/app_router.py#L229)(
```python
        elif docker_utils.find_image_by_docker_id(payload.docker_id) is None:
            raise HTTPException(status_code=404, detail="Image not found")
```
and use instead the [deployment_manager](https://github.com/Agenta-AI/agenta/blob/b544089328132ed6fac67b5ffea638a2c312af2e/agenta-backend/agenta_backend/services/deployment_manager.py#L120) and then removed the docker_utils import all together